Modules/mmapmodule.c

```diff
@@ -1659,6 +1643,23 @@ PyInit_mmap(void)
     setint(dict, "MADV_PROTECT", MADV_PROTECT);
 #endif
 #endif // HAVE_MADVISE
```

Comment:
Please add something like `if (PyErr_Occurred()) { return -1; }`. setint() silently ignore errors, but can "leak" an exception.

setint() doesn't check for PyDict_SetItemString() failure, and it doesn't report errors :-(